PProPanel/jsx/general.jsx

```diff
@@ -0,0 +1,6 @@
+if (typeof $ == 'undefined') {
```

Comment:
Well it doesn't hurt to do the extra check, if there's already a $ object defined than it won't replace it.

On the other hand we could also just get rid of it if it's really not needed. I just included it here because it was done by @bbb999 in the [PProPanel.jsx](https://github.com/Adobe-CEP/Samples/blob/master/PProPanel/PProPanel.jsx) file.